charts/neuvector/1.8.901+up1.8.9/values.yaml

```diff
@@ -0,0 +1,287 @@
+# Default values for neuvector.
+# This is a YAML-formatted file.
+# Declare variables to be passed into the templates.
+
+openshift: false
+
+registry: docker.io
+tag: 5.0.0-preview.1
+oem:
+imagePullSecrets:
+psp: false
+serviceAccount: default
```

Comment:
Does this mean its using the default service account ? It needs to use a custom one or it will break CIS benchmarks.